### PR TITLE
Release 3.0.0-rc.4

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
+
+- Updated dependency to properly style the hover state for a selected, non-disabled tab. [#41](https://github.com/blackbaud/skyux-theme/pull/41)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: These should include the repo name to be consistent with our SKYUX2 changelog:
`[blackbaud/skyux-theme#41](https://github.com/blackbaud/skyux-theme/pull/41)`

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Good catch. Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
+
+- Updated dependency to properly style the hover state for a selected, non-disabled tab. [blackbaud/skyux-theme#41](https://github.com/blackbaud/skyux-theme/pull/41)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: This is minor, but I think "for selected, non-disabled tab" is awkward and difficult to process, especially "non-disabled," which reads like a double negative. Could we change this to "for a selected tab that is active"? ... Or if changing "non-disabled" to "active" doesn't work, then how about "... for a selected tab that is not disabled"?

Anyway, please just let me know if neither of these suggestions work, and I can play around with it some more. Other than that awkward phrasing, everything looks good to me here. Thanks!

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Yeah I agree. @Blackbaud-TrevorBurch what do you think about "for a selected tab that is active" ?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I went with "... for a selected tab that is not disabled"

"Active" could mean one that is clicked on so I didn't want to go with that. Let me know if you have any other suggestions.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: Update the release date, please.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Pull request merged**